README.md

```diff
@@ -1,79 +1,88 @@
 # AnDOChecker
 
-Checks the validity of a directory with respect to the ANimal Data Organization (AnDO) specifications
+The Animal Data Organization (AnDO) is a set of specifications for the organization of a directory containing experimental data recorded in animals. It is very inspired by the BIDS specifications ( https://bids-specification.readthedocs.io ). The main difference lies in the fact that in experiments conducted with animals, it is very common to have different settings for different sessions, and more importantly for different animals (whereas research on human subjects tends to have exactly the same protocol used for all subjects, as looked after by the BIDS specifications).
 
-![version](https://img.shields.io/badge/version-1-informational)
+Futher information on Specs : https://int-nit.github.io/AnDOChecker/
 
 [![PyPI license](https://img.shields.io/pypi/l/ansicolortags.svg)](https://pypi.python.org/pypi/ansicolortags/)[![Generic badge](https://travis-ci.org/Slowblitz/BidsValidatorA.svg?branch=master)](https://shields.io/)[![made-with-python](https://img.shields.io/badge/Made%20with-Python-1f425f.svg)](https://www.python.org/)
 
-## Table of Contents
+## Installation
+### Dependencies
 
-- [AnDOChecker](#andochecker)
-  - [Table of Contents](#table-of-contents)
-  - [Use](#use)
-    - [Clone](#clone)
-    - [General usage](#general-usage)
-    - [Specific usage](#specific-usage)
-  - [Development](#development)
-    - [Testing usage](#testing-usage)
-  - [Details](#details)
+AnDOChecker requires:
 
-## Use
+- Python (>= 3.6)
+- Pip
 
-### Clone
-
-- Clone this repo to your local machine using `https://github.com/INT-NIT/AnDOChecker.git`
+### User installation
+```bash
+> Pip3 install AnDOchecker
+```
 
-### General usage
+## General usage
 
 ```term
 usage: AnDOChecker.py [-h] [-v] path
 
 positional arguments:
   path           Path to your folder
 
-optional arguments:
-  -h, --help     show this help message and exit
+optional arguments:  -h, --help     show this help message and exit
   -v, --verbose  increase output verbosity
 
 ```
 
-### Specific usage
+## Specific usage
 
 ```bash
-$Python3 checker/AnDOChecker.py tests/ds001/data/Enya
+> Python3 checker/AnDOChecker.py tests/ds001/data/Enya
```

Comment:
Hi, I think "python3 checker/AnDOChecker.py" should be replaced by the CLI script, i.e simply "AnDOChecker"... Or maybe it might be good to have one example with "python etc.", and all the other examples directly with the CLI...

Also, it should be "python", not "Python3" with a capital letter and no need for the 3 (since you said it in the dependencies...

Reply:
Updated